pages/settings/profile.tsx

```diff
@@ -105,14 +104,12 @@ export default function Settings(props: Props) {
     { value: "dark", label: t("dark") },
   ];
 
-  const usernameRef = useRef<HTMLInputElement>(null);
-  const nameRef = useRef<HTMLInputElement>(null);
-  const descriptionRef = useRef<HTMLTextAreaElement>();
-  const avatarRef = useRef<HTMLInputElement>(null);
-  const hideBrandingRef = useRef<HTMLInputElement>(null);
-  const [selectedTheme, setSelectedTheme] = useState<null | { value: string | null }>({
-    value: props.user.theme,
-  });
+  const usernameRef = useRef<HTMLInputElement>(null!);
+  const nameRef = useRef<HTMLInputElement>(null!);
+  const descriptionRef = useRef<HTMLTextAreaElement>(null!);
+  const avatarRef = useRef<HTMLInputElement>(null!);
+  const hideBrandingRef = useRef<HTMLInputElement>(null!);
```

Comment:
Added assertions since we know for sure that these [inputs exists and are referenced](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/hooks#option-1-dom-element-ref).